src/topology/instances/add_circle.lean

```diff
@@ -127,16 +186,52 @@ begin
   refl,
 end
 
+lemma lift_Ioc_coe_apply {f : 𝕜 → B} {x : 𝕜} (hx : x ∈ Ioc a (a + p)) : lift_Ioc p a f ↑x = f x :=
+begin
+  have : (equiv_Ioc p a) x = ⟨x, hx⟩,
+  { rw equiv.apply_eq_iff_eq_symm_apply,
+    refl, },
+  rw [lift_Ioc, comp_apply, this],
+  refl,
+end
+
 variables (p a)
 
+section continuity
+
 @[continuity] lemma continuous_equiv_Ico_symm : continuous (equiv_Ico p a).symm :=
 continuous_quotient_mk.comp continuous_subtype_coe
 
+@[continuity] lemma continuous_equiv_Ioc_symm : continuous (equiv_Ioc p a).symm :=
+continuous_quotient_mk.comp continuous_subtype_coe
+
+variables {x : 𝕜} (hx : (x : add_circle p) ≠ a)
```

Comment:
How about changing this to
```suggestion
variables {x : add_circle p} (hx : x ≠ a)
```
(and corresponding modifications in the lemmas below)? This would make my life easier in #17913, and I feel it's more natural overall.

Reply:
Yep, that looks strictly more general

Reply:
Sounds good; feel free to push!